build-tools/src/main/kotlin/org/jetbrains/kotlin/CompareDistributionSignatures.kt

```diff
@@ -137,4 +161,12 @@ open class CompareDistributionSignatures : DefaultTask() {
                 oldKlibSignatures - newKlibSignatures,
         )
     }
+
+    private fun isLooksLikeKotlinNativeDistribution(directory: Path): Boolean {
```

Comment:
Minor: is this a correct phrase in Engish? Maybe `looksLikeKotlinNativeDistribution`?